_projects/corpus-monodicum.md

```diff
@@ -8,4 +8,4 @@ contributors:
     - "Thomas Weber (technical concept), Notengrafik Berlin (2011–2018)"
     - "Tim Eipert (data modeling), Universität Würzburg (since 2018)"
 ---
-[Corpus monodicum](http://www.musikwissenschaft.uni-wuerzburg.de/forschung/corpus_monodicum/) is a long term edition project of historically significant sacred and secular monophonic chant repertories of the European Middle Ages with Latin texts. It chose MEI for digitizing their transcriptions in the preparation of both its print and digital online editions. As standard notation software available on the market is not well suited for capturing transcriptions of neumatic notation, the project teamed up with the music typesetting firm Notengrafik Berlin to develop the specialized browser-based notation editor mono:di that operates directly on MEI data. Since 2018 mono:di has been replaced by the editor MonodiPlus, that meets the requirements of the forthcoming online edition, and that has been developed by the software firm Olyro GmbH in cooperation with the Chair of Artificial Intelligence and Knowledge Systems, Universität Würzburg.
+[Corpus monodicum](http://www.musikwissenschaft.uni-wuerzburg.de/forschung/corpus-monodicum/) is a long term edition project of historically significant sacred and secular monophonic chant repertories of the European Middle Ages with Latin texts. It chose MEI for digitizing their transcriptions in the preparation of both its print and digital online editions. As standard notation software available on the market is not well suited for capturing transcriptions of neumatic notation, the project teamed up with the music typesetting firm Notengrafik Berlin to develop the specialized browser-based notation editor mono:di that operates directly on MEI data. Since 2018 mono:di has been replaced by the editor MonodiPlus, that meets the requirements of the forthcoming online edition, and that has been developed by the software firm Olyro GmbH in cooperation with the Chair of Artificial Intelligence and Knowledge Systems, Universität Würzburg.
```

Comment:
```suggestion
[Corpus monodicum](https://corpus-monodicum.de) is a long term edition project of historically significant sacred and secular monophonic chant repertories of the European Middle Ages with Latin texts. It chose MEI for digitizing their transcriptions in the preparation of both its print and digital online editions. As standard notation software available on the market is not well suited for capturing transcriptions of neumatic notation, the project teamed up with the music typesetting firm Notengrafik Berlin to develop the specialized browser-based notation editor mono:di that operates directly on MEI data. Since 2018 mono:di has been replaced by the editor MonodiPlus, that meets the requirements of the forthcoming online edition, and that has been developed by the software firm Olyro GmbH in cooperation with the Chair of Artificial Intelligence and Knowledge Systems, Universität Würzburg.
```